project-resources/PROJECT_FUNDS.md

```diff
@@ -0,0 +1,24 @@
+# OpenJS Foundation Project-specific Initiative Funds
+
+The OpenJS Foundation provides a set of monetary funds for projects of the OpenJS Foundation. 
+These funds are designed to support collaboration and incentive project-specific initiatives that have high-impact within the individual project's scope.
+
+Alongside these lines, the fund exist to support the development of these 
+initiatives and give visibility/recognition to the champions of these project-specific initiatives, highligthing the individuals behind them.
+
+## F.A.Q.
+
+#### How might a project request for funds?
+
+Funds should be requested on an ad-hoc basis through a PR requesting the funds. The requests do not need to follow a specific template,
+but the description should contain relevant information regarding the project initiative, 
+and any information that might be useful to highlight the need of funds and their benefits.
+
+#### How these requests get evaluated?
+
+The requests should go through the OpenJS Foundation CPC (Cross Project Council) and must have approval from the CPC members. 
```

Comment:
It would be interesting to know if CPC voting must be unilateral, or how exactly the voting works.

I would also suggest for us to have a Markdown file (Basically a table) like we have for Travel Funds to keep documented previous "approved project funds requests".

Reply:
Do we have existing approval/voting processes we can fall back on?

Reply:
The only similar process I'm aware of is the travel fund...

Reply:
> It would be interesting to know if CPC voting must be unilateral, or how exactly the voting works.

_"The request must be made to the CPC for evaluation. The CPC evaluates the request following its [decision-making process](https://github.com/openjs-foundation/cross-project-council/blob/a49a8d2e04716525116c0ef787e1187d4dcd9e3e/CPC-CHARTER.md#section-9-decision-making) and makes a recommendation to the Board."_